scripts/build_pkg.sh

```diff
@@ -39,6 +39,7 @@ fi
 DEPS_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/deps/usr
 FWUP_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/fwup-staging/usr
 PKG_CONFIG_PATH=$DEPS_INSTALL_DIR/lib/pkgconfig
+export ChocolateyInstall=$DEPS_INSTALL_DIR/chocolatey
```

Comment:
Does `ChocolateyInstall` need to be exported? If not, then I'd prefer it to be `CHOCOLATEY_DIR` to be more consistent with the other variables.